core/client/src/main/java/alluxio/hadoop/HdfsFileInputStream.java

```diff
@@ -390,4 +390,18 @@ public void seek(long pos) throws IOException {
   public boolean seekToNewSource(long targetPos) throws IOException {
     throw new IOException(ExceptionMessage.NOT_SUPPORTED.getMessage());
   }
+
+  @Override
```

Comment:
can you document here that the base class implementation of `skip(long)` method is not efficient? Therefore we override and implement `skip` in this class? In this way, we may prevent removing the overriding implementation by accident.